app/views/edit/deployment-config.html

```diff
@@ -356,6 +356,11 @@ <h3 class="with-divider">Environment Variables</h3>
                         </div>
                       </div>
 
+                      <pause-rollouts-checkbox
+                          deployment="updatedDeploymentConfig"
+                          always-visible="true">
```

Comment:
I thought you might want to pause/resume rollouts in this editor for other reasons (e.g. to keep image change triggers from firing)